ext/standaloneusers/settings/standaloneusers.setting.php

```diff
@@ -16,13 +16,19 @@
   // Example value: 'TOTP', not 'Civi\Standalone\MFA\TOTP'
   'standalone_mfa_enabled' => [
     'name'        => 'standalone_mfa_enabled',
+    'serialize'   => CRM_Core_DAO::SERIALIZE_JSON,
     'group'       => 'standaloneusers',
     'type'        => 'String',
```

Comment:
Wasn't working for me because the setting was coming back as a string rather than PHP array. Fixed it with:
```suggestion
    'group'       => 'standaloneusers',
    'type'        => 'Array',
```